### DISCUSSION TITLE
Make a PR to vim

### DISCUSSION THREAD

**Discussion by tcheukueppo** [trizen/sidef#109] [Ideas]
Hello,

I think we should add a colorscheme to vim for the sidef programming language, what do you think?

**Comment by trizen** [comment]
I think the Ruby colorscheme works pretty well in most cases.

But we can definitely try to modify the Ruby colorscheme for Sidef, but, unfortunately, I don't have the time at the moment to do so.

**Comment by tcheukueppo** [comment]
Cool, I'll be working on it whenever possible.
